api/migrations/40-search-tasks-n-ctfs.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX ON ctfnote.ctf (title);
+CREATE INDEX ON ctfnote.task (title);
```

Comment:
The popup says:
> search for title or description
but you can only search for the title of a task or CTF, but not the actual description. Do we want to index the description too?

Reply:
Maybe we can use description and category later =)